apps/web/src/pages/Applications/ApplicationProfilePage/ApplicationProfilePage.tsx

```diff
@@ -75,16 +100,195 @@ export const getPageInfo: GetApplicationPageInfo = ({
+              id: "CPHTk9",
+              description:
+                "Error message when a user does not meet a pools language requirements",
+            },
+            { requirements },
+          ),
+        );
+      }
+    }
+
+    return false;
+  };
+
+  const handleUpdate: SectionProps["onUpdate"] = (userId, userData) => {
+    return executeUpdateMutation({
+      id: userId,
+      user: userData,
+    }).then((res) => res.data?.updateUserAsUser);
+  };
+
+  const sectionProps = {
+    user,
+    isUpdating,
+    onUpdate: handleUpdate,
+  };
 
-  return <Heading data-h2-margin-top="base(0)">{pageInfo.title}</Heading>;
+  return (
+    <>
+      <Heading data-h2-margin-top="base(0)">{pageInfo.title}</Heading>
+      <p>
+        {intl.formatMessage({
+          defaultMessage:
+            "This step includes all of your profile information required for this application. Each section has a series of required fields that must be completed to move on to the next step. Completing this profile information will only happen once, and the information you provide here will auto-populate on applications you submit moving forward.",
+          id: "D9Elm9",
+          description: "Application step to complete your profile, description",
+        })}
+      </p>
+      <div
+        data-h2-margin="base(x2, 0)"
+        data-h2-display="base(flex)"
+        data-h2-flex-direction="base(column)"
+        data-h2-gap="base(x1, 0)"
+      >
+        <ErrorSummary user={user} application={application} />
+        <PersonalInformation {...sectionProps} />
+        <WorkPreferences {...sectionProps} />
+        <DiversityEquityInclusion {...sectionProps} />
+        <GovernmentInformation {...sectionProps} />
+        <LanguageProfile {...sectionProps} application={application} />
+      </div>
+      <Separator
+        orientation="horizontal"
+        data-h2-background-color="base(gray.lighter)"
+        data-h2-margin="base(x2, 0)"
+        decorative
+      />
+      <div
+        data-h2-display="base(flex)"
+        data-h2-gap="base(x.25, x.5)"
+        data-h2-flex-wrap="base(wrap)"
+        data-h2-flex-direction="base(column) l-tablet(row)"
+        data-h2-align-items="base(flex-start) l-tablet(center)"
+      >
+        <form onSubmit={handleNavigation}>
+          <Button
+            type="submit"
+            color="primary"
+            mode="solid"
+            disabled={submitting}
+          >
+            {intl.formatMessage({
+              defaultMessage: "Let's go!",
+              id: "r6z4HM",
+              description: "Link text to begin the application process",
+            })}
+          </Button>
+        </form>
+        <Link
+          type="button"
+          mode="inline"
+          color="secondary"
+          href={applicantDashboard ? paths.dashboard() : paths.myProfile()}
+        >
+          {intl.formatMessage({
+            defaultMessage: "Save and quit for now",
```

Comment:
How can a user save an application if the application hasn't been started (based on the adjacent button that begins an application process: **Link text to begin the application process**)?

Reply:
I think I wrote that link description.  It's not really very accurate since (as you pointed out) the application has already started.  It should probably be something like "link to move to the next step of the application".  

It's also correct (as we discussed in scrum) that this button doesn't actually save.  :confused:

Reply:
While I'm still not a fan of this button or the link text since it does not provide any tangible value (seems just like the long way to get back to your dashboard), it should be functioning as described by the designers.

Reply:
I agree with @esizer. Using **Save** on adjacent buttons (**Save and continue**, **Save and quit for now**) could lead to decision paralysis.